tests/threadpooltest.py

```diff
@@ -271,9 +281,19 @@ def testWrapper(self):
         def accounting(i):
             counts[i] += 1
         qsequence = [(mean,"mean"), (simple,"simple")]
+
+        # Ensure full buffering before test start, to make sure
+        # proper batch priorization is taking place, as it won't
+        # under light load
+        barrier = Event()
+        for i in range(32):
```

Comment:
where does this 32 come from?

Reply:
Max number of threads I expect to be running this thread. Totally a guess.